### DISCUSSION TITLE
vue2 项目如何 使用NutUI 3.x 的UI界面

### DISCUSSION THREAD

**Discussion by wjq152** [jdf2e/nutui#1490] [Q&A]
个人感觉 NutUI 3x 风格好看，但是公司项目基本是vue2， NutUI 2x未来会更新UI风格吗

**Comment by richard1015** [comment]
> 个人感觉 NutUI 3x 风格好看，但是公司项目基本是vue2， NutUI 2x未来会更新UI风格吗

不会
